src/main/resources/sql/1_CREATE_TABLES.sql

```diff
@@ -0,0 +1,75 @@
+CREATE TABLE USER_TYPE (
+                        	IDT_USER_ID INTEGER PRIMARY KEY NOT NULL, USER_TYPE VARCHAR(30)
+);
+
+CREATE TABLE PERSON (
+                     	IDT_PERSON_ID INTEGER PRIMARY KEY NOT NULL,
```

Comment:
não é assim que definimos primary key
IDT_PERSON_ID INTEGER NOT NULL

ai depois
CONSTRAINT PERSON_PK PRIMARY KEY (IDT_PERSON)